src/components/ClusterNodes/NodeList.tsx

```diff
@@ -633,6 +634,7 @@ export default function NodeList({ imageList, isSuperAdmin, namespaceList }: Clu
 
     const openTerminal = (clusterData): void => {
         setTerminalCluster(clusterData.name)
```

Comment:
Both `setTerminalCluster`and  `setSelectedNode`are setting the same name then is there any need to maintain one extra state?

Reply:
removed setTerminalCluster useState as we can't change cluster on this page